src/sentry/assistant/guides.py

```diff
@@ -3,6 +3,24 @@
 from django.utils.translation import ugettext_lazy as _
 
 
+# Guide Schema
+# id (text, required): unique id
+# cue (text, optional):  The text used to prompt the user to initiate the guide.
```

Comment:
I don't think any of the fields we have can be optional? (in the sense you may just omit it instead of making it [] or None)

Reply:
Ah, that's true. I'll fix.